etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/RemoveMessageTypeStructure.java

```diff
@@ -0,0 +1,36 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom;
+
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.FhirResource;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.RuleExecutionException;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.CustomFhirTransformation;
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Map;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Coding;
+
+/** Removes the Message Structure (MSH-9.3) from the Message Type (MSH-9). */
+public class RemoveMessageTypeStructure implements CustomFhirTransformation {
+
+    @Override
+    public void transform(FhirResource<?> resource, Map<String, String> args)
+            throws RuleExecutionException {
+        try {
+            Bundle bundle = (Bundle) resource.getUnderlyingResource();
+            Coding coding = HapiHelper.getMessageTypeCoding(bundle);
+            String display = coding.getDisplay();
+            String delimiter = "^";
+            String[] displayArray = display.split("\\" + delimiter);
```

Comment:
1. Does display need a null check?

2. If you use something like the below you can take display and get the ArrayList you want back in one line.  This would allow you to remove/refactor some of the following lines 

```suggestion
                        ArrayList displayArray = Arrays.stream(display.split("\\" + delimiter)).collect(Collectors.toCollection(ArrayList::new));
```